### PR TITLE
Make SPAKE2 more explicit

### PR DESCRIPTION
**Pull request by pthatcherg**
Hopefully fixes https://github.com/webscreens/openscreenprotocol/issues/226

***
<a href="https://pr-preview.s3.amazonaws.com/webscreens/openscreenprotocol/pull/227.html" title="Last updated on Oct 1, 2019, 6:20 PM UTC (3846e9c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/webscreens/openscreenprotocol/227/3afb5c8...3846e9c.html" title="Last updated on Oct 1, 2019, 6:20 PM UTC (3846e9c)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -711,14 +711,18 @@ The client acts as Alice, the server acts as Bob.
 Issue: What do "client" and "server" mean here?
 
 The messages used in this authentication method are: [=auth-spake2-need-psk=],
-[=auth-spake2-message=], [=auth-spake2-confirmation=] and [=auth-status=].
-SPAKE2 describes in detail how [=auth-spake2-message=] and [=auth-spake2-confirmation=]
-are computed.
+[=auth-spake2-handshake=], [=auth-spake2-confirmation=] and [=auth-status=].
+SPAKE2 describes in detail how [=auth-spake2-handshake=] and [=auth-spake2-confirmation=]
+are computed.  The values `A` and `B` used in SPAKE2 are the agent certifiate
```
Comment (baylesj) [inline index.bs]: What messages are A and B actually used in? Is that the same as the certificate fingerprint broadcasted as part of mDNS?

**Comment by baylesj** [comment]
LGTM, would like some explicit clarification on the 'A' and 'B' fields. Also remember  to run bikeshed and update the HTML files.

File: index.bs (hunk 1 above)
Comment (mfoltzgoogle) [reply index.bs]: [=agent certificate=] (fixing typo)

File: index.bs [hunk 2]
```diff
@@ -711,14 +711,18 @@ The client acts as Alice, the server acts as Bob.
...
+fingerprints of the client and server, respectively. `K` is the PSK presented to
+the user.  `S` and `T` from SPAKE2 are put into the `random` field of
+[=auth-spake2-handshake=].  `F` are from SPAKE2 are put into the
```
Comment (mfoltzgoogle) [inline index.bs]: The sentence beginning with 'F' doesn't make a ton of sense to me. :-P

File: index.bs (hunk 1 above)
Comment (mfoltzgoogle) [reply index.bs]: A and B are used to compute the auth-spake2-confirmation value.

**Comment by mfoltzgoogle** [comment]
I went ahead and fixed up the gammar, merging.

**Pull request merged**